src/olgm/gm/MapIcon.js

```diff
@@ -112,7 +112,15 @@ class MapIcon extends MapElement {
 
     const panes = this.getPanes();
     if (panes) {
-      panes.markerLayer.appendChild(canvas);
+      let pane = this.get('pane');
+      if (pane) {
+        pane = panes[pane];
```

Comment:
Trivial comment: we could also validate that the pane exists.  In other word, fallback to the markerLayer pane if `pane` is  not defined there because of the use of a non existant pane name.

Reply:
Updated to handle non-existent pane name.